docker-compose.yml

```diff
@@ -1,5 +1,4 @@
 version: '3'
-
 services:
   postgres:
     image: postgres
```

Comment:
please, set a specific version for this image
I hope you have no problems with a bug with a new version of this image and keep debugging for hours, to find out the cause of the error.